build.gradle

```diff
@@ -92,12 +92,13 @@ dependencies {
     implementation 'com.google.guava:guava:31.1-jre'
     testImplementation group: 'junit', name: 'junit', version: '4.13.2'
     testImplementation 'org.spockframework:spock-core:2.0-groovy-3.0'
-    testImplementation 'org.codehaus.groovy:groovy:3.0.16'
+    testImplementation 'org.codehaus.groovy:groovy:3.0.17'
```

Comment:
Might as well update Groovy while I'm here. This was released at the end of March.